### ISSUE TITLE
Enforcing TSL 1.2 and later

### ISSUE THREAD

**Issue by elad-ep** [aws/aws-iot-device-sdk-cpp-v2#118]
Hi, how do I enforce TLS > 1.2? 

I found this article for AWS c++ SDK:
https://docs.aws.amazon.com/en_us/sdk-for-cpp/v1/developer-guide/enforcing-tls.html

Is there any configuration for the IoT sdk?

**Comment by rccarper** [comment]
Hello, it looks like we have support for this in our underlying IO library, but it's not currently exposed to our C++ CRT, which this SDK uses.  I am looking into making the necessary changes.

**Comment by rccarper** [comment]
Hello, we are still working on getting this change in.  We should have it available sometime (hopefully early) next week.

**Comment by elad-ep** [comment]
thanks for the update! after completing this work could you please create a release version (v1.6.1?) that includes this enhancement?

**Comment by rccarper** [comment]
Yes, we have one more change to make/merge, and then we should be able to cut a new release within the next day or two. :)

**Comment by elad-ep** [comment]
I took the later version v1.7.0 but the TLS exposure feature is not there (no WithMinimumTlsVersion in MqttClient.h). I went back to v1.6.2 and the TLS feature is there... 
was is dropped from v1.7.0 by mistake?

**Comment by rccarper** [comment]
I apologize for the issue there!  We just cut a new release (v1.7.1) which should have the fix back in it.
